src/envdrift/integrations/dotenvx.py

```
@@ -556,6 +574,151 @@ def _run(
+            DotenvxFilenameError: If the filename matches a known problematic pattern.
+        """
+        filename = file_path.name.lower()
+        for pattern in PROBLEMATIC_FILENAME_PATTERNS:
+            if re.match(pattern, filename, re.IGNORECASE):
+                raise DotenvxFilenameError(
+                    f"Cannot encrypt/decrypt '{file_path.name}': "
+                    f"dotenvx has a known bug on Windows where this filename causes "
+                    f"'Input string must contain hex characters in even length' error. "
+                    f"Workaround: Rename the file (e.g., '.env.localenv' or '.env.dev') "
+                    f"before encryption. See: https://github.com/dotenvx/dotenvx/issues/XXX"
+                )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)